setup.py

```diff
@@ -942,17 +943,17 @@ def __init__(self, attrs=None):
         with open(SALT_LONG_DESCRIPTION_FILE, encoding="utf-8") as f:
             self.long_description = f.read()
         self.long_description_content_type = "text/x-rst"
-        self.python_requires = ">=3.5"
+        self.python_requires = ">=3.6"
         self.classifiers = [
             "Programming Language :: Python",
             "Programming Language :: Cython",
             "Programming Language :: Python :: 3",
             "Programming Language :: Python :: 3 :: Only",
-            "Programming Language :: Python :: 3.5",
             "Programming Language :: Python :: 3.6",
```

Comment:
```suggestion

```

Reply:
I thought about it, and originally did, but added 3.6 back due to answer to https://saltstack.slack.com/archives/C02D6MGTGF3/p1671162285865489, were running old tests.